tests/models/revision.py

```diff
@@ -45,18 +41,17 @@ def test_max_revision(revision, project0_nongnu, store0):
 def test_revision_incr(store0):
     """Tests revision is incremented when units change."""
     previous_revision = Revision.get()
-    db_unit = _update_translation(store0, 0, {'target': [u'Fleisch']},
-                                  sync=False)
-
+    db_unit = store0.units.exclude(target_f="").first()
```

Comment:
I don't think it matters, just curious why the qs was being filtered at all. I am happy leaving this as is provided Travis passes.